README.md

```diff
@@ -1,5 +1,97 @@
-The following tasks must be completed:
+# Tasks
 
-1. Write a method that takes an array of String elements as an input and returns Map <String, Integer>, where String is the value from the array and Integer is the number of entries in the array.
-2. Write an Iterator over an array.
-3. Write a method to build a frequency dictionary of the letters of the English alphabet.
+## 1. Lessons Getter
+
+Implement the program which gets timetable and returns `set` of all lessons without duplicates. Timetable is
+a `Map<String, List<String, String>>`, where the `key` - the day of the week and the `value` - `List<String> `of lessons
+for that day.
+
+```
+public class LessonsGetter {
+    public Set<String> getLessons(Map<String, List<String>> timetable){
+    }
+}
+```
+
+### Example
+
+**Input**
+
+```
+Modnay - English
+Tuesday - Mathematics
+Wednesday - English, Chemistry
+Thursday - Literature, Mathematics
+Friday - Physics
+```
+
+**Output**
+
+```
+English, Mathematics, Chemistry, Literature, Physics
+```
+
+## 2. Map from keys Creator
+
+Implement the program which gets `Map<String, Integer>` as parameter and returns a new `Map<Integer, Set<String>>`,
```

Comment:
I don't understand why you use `Map<String, Integer>` as parameter if you don't use values from map.

Reply:
the idea was to use keySet()